tests/Validity.hs

```diff
@@ -257,7 +265,7 @@ instance (GenUnchecked k, GenUnchecked v) => GenUnchecked (HashMap k v) where
 
 -- It turns out it's almost impossible to write this instance without using internals.
 -- This is good-enough for now.
-instance (Eq k, Hashable k, GenValid k, GenValid v) =>
+instance (Eq k, Hashable k, GenValid k, Eq v, GenValid v) =>
```

Comment:
This `Eq` looks like an artifact of the temporary change. Probably don't need it?